### ISSUE TITLE
Python 3: "String argument without an encoding" error with non-empty query string

### ISSUE THREAD

**Issue by kogans** [grisha/mod_python#41]
Output:

Mod_python error: "PythonHandler mod_python.publisher"

Traceback (most recent call last):

  File "/usr/lib64/python3.4/site-packages/mod_python/apache.py", line 398, in HandlerDispatch
    result = obj(req)

  File "/usr/lib64/python3.4/site-packages/mod_python/publisher.py", line 222, in handler
    published = publish_object(req, object)

  File "/usr/lib64/python3.4/site-packages/mod_python/publisher.py", line 445, in publish_object
    req.form = util.FieldStorage(req, keep_blank_values=1)

  File "/usr/lib64/python3.4/site-packages/mod_python/util.py", line 250, in __init__
    self.add_field(pair[0], pair[1])

  File "/usr/lib64/python3.4/site-packages/mod_python/util.py", line 398, in add_field
    item = StringField(value)

  File "/usr/lib64/python3.4/site-packages/mod_python/util.py", line 159, in __new__
    return bytes.__new__(self, value)

TypeError: string argument without an encoding

**Comment by lovasoa** [comment]
I have the same error.
I fixed it by manually editing [/usr/lib64/python3.4/site-packages/mod_python/util.py, line 159](https://github.com/grisha/mod_python/blob/master/lib/python/mod_python/util.py#L159), changing
```
            return bytes.__new__(self, value)
```
to
```
            return bytes.__new__(self, value, "utf8")
```
and restarting apache.

But the same code wouldn't work in python2. So I'm not sure I can do a pull request with this.

**Comment by lovasoa** [comment]
The python2 case is handled on [line 114](https://github.com/grisha/mod_python/blob/master/lib/python/mod_python/util.py#L114). So I'm doing a pull request.

**Comment by grisha** [comment]
I'm kind of curious why this wasn't a problem before, is this a Python 3.4 change?

**Comment by lovasoa** [comment]
@grisha : No, it looks like it's been there for a long time: https://github.com/python/cpython/blame/dcaef9120f263f1f54820a7219a48f434b162c4b/Objects/bytesobject.c#L3167

**Comment by grisha** [comment]
Yep, @lovasoa I was just looking at the exact same line... You'd think this would have surfaced sooner. Anyway, thanks for the PR, I'll take a look at it soon - I want to see if the same thing happens with earlier versions of Python.

**Comment by lovasoa** [comment]
I think there is a problem with my PR. I wrote "utf8" because it seemed obvious, but a simple
```python
def repeat(req, param):
   return param
```
called with `http://localhost/test.py/repeat?param=é` exposes the problem.

**Comment by lovasoa** [comment]
Fixed by #44
